documentation/get-started/index.en.md

```diff
@@ -43,18 +47,15 @@ QField is an open-source application that allows painless fieldwork and synchron
     Download the latest <a href="https://qfield.org/get_latest/?platform=macos" target="_blank">MacOS
     version</a> (Universal).
 
-QField allows you to efficiently work on your GIS data outdoor.
-
-QField's optimized user interface hides the full power of
-[QGIS](https://qgis.org) under the hood.
+QField is available for Android and iOS devices and can also be accessed on the Desktop (Windows, macOS and Linux).
+Use QField for optimised data collection outside in any place on earth.
+Its optimized user interface hides the full power of [QGIS](https://qgis.org) under the hood.
```

Comment:
```suggestion
Its optimized user interface exposes through a an easy-to-use front-end that gets out of the way the full power of [QGIS](https://qgis.org) under the hood.
```

Reply:
Rational: I feel "hiding" is a bit negative. We don't hide features, we channel through through a much cleaner interface that's friendlier and won't overwhelm you on first approach.